platform/swift/source/replay/SessionReplayTarget.swift

```diff
@@ -41,14 +42,36 @@ extension SessionReplayTarget: CapturePassable.SessionReplayTarget {
     }
 
     func captureScreenshot() {
-        // TODO: Implement
-        //        DispatchQueue.main.async { [weak self] in
-        //            self?.queue.async {
-        //                self?.logger?.logSessionReplayScreenshot(
-        //                    screen: SessionReplayCapture(data: Data()),
-        //                    duration: 0
-        //                )
-        //            }
-        //        }
+        DispatchQueue.main.async {
+            guard let window = UIApplication.shared.sessionReplayWindows().first else {
+                // According to the documentation for
+                // `CapturePassable.SessionReplayTarget.captureScreenshot()`, this method must emit
+                // a screenshot log. Without this, the Rust logger will ignore any further screenshot
+                // capture actions.
+                // To handle cases where no window is available, an assertion is included here.
+                // TODO: Add a log to inform the Rust layer if the screenshot log emission fails, allowing it
+                // to proceed to the next screenshot (if any).
+                assertionFailure("no window to take screenshot of")
```

Comment:
If that’s the case might be better to send an empty data object back for now, wdyt?